packages/basic/index.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+import { existsSync } from 'node:fs'
+import { readFile } from 'node:fs/promises'
+import { resolve } from 'node:path'
+
+import { ConfigManager } from '@platformatic/config'
+
+import { packageJson, schema } from './lib/schema.js'
+import { ServerStackable } from './lib/server.js'
+
+const validFields = [
+  'main',
+  'exports',
+  'exports',
+  'exports#node',
+  'exports#import',
+  'exports#require',
+  'exports#default',
+  'exports#.#node',
+  'exports#.#import',
+  'exports#.#require',
+  'exports#.#default',
+]
+
+const validFilesBasenames = ['index', 'main', 'app', 'application', 'server', 'start', 'bundle', 'run', 'entrypoint']
+
+async function parsePackageJson (root) {
+  let entrypoint
+  let packageJson
+  let hadEntrypointField = false
```

Comment:
If this means that the service is a runtime entrypoint, you can get this info from "context.isEntrypoint"